colorcoder/colorInterface.java

```diff
@@ -0,0 +1,7 @@
+package colorcoder;

```

Comment:
colorInterface - interface name should have pascal convention ( ColorInterface)

When you name interface don't suffix language building block